test/integration/job/job_test.go

```diff
@@ -1068,6 +1073,7 @@ func TestBackoffLimitPerIndex_Reenabling(t *testing.T) {
 	validateIndexedJobPods(ctx, t, clientSet, jobObj, sets.New(1, 2), "", ptr.To("0"))
 
 	// Disable the feature
+	featuregatetesting.SetFeatureGateEmulationVersionDuringTest(t, feature.DefaultFeatureGate, utilversion.MustParse("1.32"))
```

Comment:
Even though this works, I think it would be nicer to do this at the beginning of the test.

Reply:
Sure, done.